stdlib/uchar.mli

```diff
@@ -143,6 +143,20 @@ val utf_decode_invalid : int -> utf_decode
     smaller or equal to [4] (this is not checked by the module). The
     resulting decode has {!rep} as the decoded Unicode character. *)
 
+val utf_8_decode_length_of_byte : char -> int
+(** [utf_8_decode_length_of_byte byte] is the number of bytes that a
+    UTF-8 decode starting with byte [byte] would consume. This is either
```

Comment:
Since the descriptions above introduce the concept of valid and invalid decode, I would suggest to mention that this is the number of byte a  *valid* UTF-8 decode would consume.

Reply:
I think it would be slightly misleading wording entailing that `byte` should be a valid UTF-8 starter. The function is agnostic to whether the decode will be valid or not. I think the subsequent sentence makes things clear.

Reply:
But I agree the current formulation is not optimal. Let me change that.